modules/lang-groovy/src/test/java/org/elasticsearch/script/groovy/GroovySecurityTests.java

```diff
@@ -123,6 +123,13 @@ public void testEvilGroovyScripts() throws Exception {
         }
     }
 
+    public void testGroovyScriptsThatThrowErrors() throws Exception {
+        assertFailure("assert false, \"msg\";", AssertionError.class);
+        assertFailure("def foo=false; assert foo;", AssertionError.class);
```

Comment:
In order to do that, we have to add permissions for the `java.lang.NoClassDefFoundError` in ClassPermission.java. Are we sure we want to add permissions for that class only to add a test case?